Documents/Help/Changelog.md

```diff
@@ -5,6 +5,7 @@
 =======
 
 ## 2022-11-xx - Build 2211 - November 2022 <!--Possible August or September release?-->
+* Implemented [#761](https://github.com/Krypton-Suite/Standard-Toolkit/issues/761), Can TFM also include .NET Framework 4.8.1? (**Note:** This only applies to users of Visual Studio 2022)
```

Comment:
Should state "Currently" as it will become a public "full redist" (As that is what is used by vs 2022 anyway)

Reply:
Resolved